offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/copy/CopyProductView.groovy

```diff
@@ -0,0 +1,41 @@
+package life.qbic.portal.offermanager.components.product.copy
+
+import com.vaadin.event.MouseEvents
+import com.vaadin.ui.Button
+import life.qbic.datamodel.dtos.business.ProductId
+import life.qbic.datamodel.dtos.business.services.Product
+import life.qbic.portal.offermanager.components.product.MaintainProductsController
+import life.qbic.portal.offermanager.components.product.create.CreateProductView
+
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
+ *
```

Comment:
```suggestion
 * This class represents the GUI for copying a product
 *
 * The view is similar to the {@link CreateProductview} and updates the view to fit the copy product use case
 *
 * @since: 1.0.0
 *
```

Reply:
Caveman no need for puny documentation 🐒